clang/include/clang/Driver/Options.td

```diff
@@ -7511,6 +7518,13 @@ def fsyclbin : Flag<["-"], "fsyclbin">, Alias<fsyclbin_EQ>,
   AliasArgs<["executable"]>;
 } // let Group = sycl_Group
 
```

Comment:
Can you please add a comment here to differentiate SYCL options and SYCLRTCOnly option.